### PR TITLE
Update three_lauren_amy_dan_jason_chelle.md

### PR DESCRIPTION
**Pull request by chelletuerk**

### CODE REVIEW

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+Sending out the reading a day in advance.  Helps facilitate discussion.
+
+Need to find an article that is a polar opposite view on the material presented in the original gear up.
+
+
+# Session 2    3/31/17
+
+### Intro 5 min
+Consider creating a 'spectrum of opinion' as to where one stands on the issue being discussed  (similar to when we lined up on the wall during the original AI gear up)   3-5 minutes
+#### ask for 'why'
```
Comment (case-eee) [inline m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:33]: will students write during this time? or will they share to the group?

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+Would have like more time to mull it over solo and writing time.
+
+Sending out the reading a day in advance.  Helps facilitate discussion.
+
+Need to find an article that is a polar opposite view on the material presented in the original gear up.
+
+
+# Session 2    3/31/17
+
+### Intro 5 min
```
Comment (case-eee) [inline m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:31]: 👍

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+
+Environmental impact of making and maintaining robots?
+
+### GENERAL SESSION OUTLINE REFORMATS:
+
+Would have like more time to mull it over solo and writing time.
+
+Sending out the reading a day in advance.  Helps facilitate discussion.
+
+Need to find an article that is a polar opposite view on the material presented in the original gear up.
```
Comment (case-eee) [inline m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:26]: any progress finding an article?

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md [hunk 4]
```diff
@@ -1,3 +1,51 @@
...
+
+What is it going to do to the quality of life of not the two ends of the financial spectrum, but the middle-class?
+
+Environmental impact of making and maintaining robots?
+
+### GENERAL SESSION OUTLINE REFORMATS:
+
+Would have like more time to mull it over solo and writing time.
+
+Sending out the reading a day in advance.  Helps facilitate discussion.
```
Comment (case-eee) [inline m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:24]: who's going to send the reading in advance? what specific articles will they be reading?

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md [hunk 5]
```diff
@@ -1,3 +1,51 @@
 ## Group Three's Session Plan
 
 M4 Student Leaders: Lauren, Amy, Dan, Jason, Chelle
+
+
+# Session 1   3/17/17
+
+### ADDITIONAL TOPICS FOR DISCUSSION THAT WERE GLOSSED OVER:
```
Comment (case-eee) [inline m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:8]: love all these topics! do you think you even need to watch the movie? are the articles/reflection enough to have a meaningful conversation about this topic?

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md (hunk 3 above)
Comment (DanGrund) [reply m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:26]: I found two, not sure why they aren't merged. 
http://www.economicshelp.org/blog/6717/economics/the-luddite-fallacy/ a brief introduction on the Luddite Fallacy, and the wikipedia on the Post-Scarcity economy theory https://en.wikipedia.org/wiki/Post-scarcity_economy

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md (hunk 4 above)
Comment (DanGrund) [reply m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:24]: I'll take care of the reading send out, with the articles from the original gear up, as well as the two opposing view points expressed below

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md (hunk 1 above)
Comment (DanGrund) [reply m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:33]: a little bit of time for writing and reflection, but mostly discussion

File: m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md (hunk 3 above)
Comment (case-eee) [reply m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md:26]: hmm interesting - feel free to add these before i merge this 👍

**Pull request closed without merging**